pkg/apis/k0s/v1beta1/api.go

```diff
@@ -64,6 +65,14 @@ type APISpec struct {
 	// List of additional addresses to push to API servers serving the certificate
 	// +listType=set
 	SANs []string `json:"sans,omitempty"`
+
+	// The expiration duration of the CA certificate
+	// +kubebuilder:default="87600h"
+	CAExpiry string `json:"caExpiry,omitempty"`
```

Comment:
Wouldn't it be better to use `metav1.Duration` for these fields instead of `string`?

Reply:
Updated